src/types/markers.js

```diff
@@ -498,6 +498,7 @@ export type NetworkPayload = {|
   // It's always absent in Firefox < 98 because we couldn't capture private
   // browsing data back then.
   isPrivateBrowsing?: boolean,
+  httpVersion?: string,
```

Comment:
replace `string` with `"h3" | "h2" | "http/1.1" | "http/1.0"` so that the types can help us with later changes.
Also this documents the network payload so that other users of the frontend can also see what is expected by our code.